tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -552,3 +552,39 @@ def test_monitoring_delete_pvc(self):
         assert prometheus_health_check(), (
             "Prometheus cluster health is not OK"
         )
+
+    @pytest.mark.polarion_id("OCS-1535")
+    def test_monitoring_shutdown_mgr_pod(self, pods):
+        """
+        Montoring backed by OCS, bring mgr down(replica: 0) for some time
+        and check ceph related metrics
+        """
+        # Check ceph metrics available
+        assert check_ceph_metrics_available(), (
+            "failed to get results for some metrics before Downscaling deployment mgr to 0"
+        )
+
+        # Reduce mgr pod deployments to replicas=0
+        oc = ocp.OCP(
+            kind=constants.DEPLOYMENT, namespace='openshift-storage'
+        )
+
+        # Downscaling deployment mgr to 0
+        mgr_deployments = oc.get(selector=constants.MGR_APP_LABEL)['items']
+        mgr = mgr_deployments[0]['metadata']['name']
+        log.info(f"Downscaling deployment {mgr} to 0")
+        oc.exec_oc_cmd(f"scale --replicas=0 deployment/{mgr}")
+
+        log.info('wait 5 min')
+        time.sleep(300)
```

Comment:
is there anything we can wait for? such as the count of MGRs?

Reply:
@ebenahar 
After downscaling mgr deployment, rook-ceph-mgr-a pod disappear from the output of  "oc get pods" command.